presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestOraclePoolIntegrationSmokeTest.java

```diff
@@ -13,82 +13,16 @@
  */
 package io.prestosql.plugin.oracle;
 
-import io.prestosql.testing.AbstractTestIntegrationSmokeTest;
-import io.prestosql.testing.MaterializedResult;
 import io.prestosql.testing.QueryRunner;
 import io.prestosql.tpch.TpchTable;
-import org.testcontainers.shaded.com.google.common.collect.ImmutableList;
-import org.testng.SkipException;
-import org.testng.annotations.AfterClass;
-import org.testng.annotations.Test;
-
-import static io.prestosql.spi.type.VarcharType.VARCHAR;
-import static io.prestosql.testing.assertions.Assert.assertEquals;
-import static io.prestosql.tpch.TpchTable.*;
-import static io.prestosql.tpch.TpchTable.REGION;
-import static org.assertj.core.api.Assertions.assertThat;
 
 public class TestOraclePoolIntegrationSmokeTest
-        extends AbstractTestIntegrationSmokeTest
+        extends BaseOracleIntegrationSmokeTest
```

Comment:
the `Oracle IntegrationTest cleanups` should not be a separate commit
(we should not add `TestOraclePoolIntegrationSmokeTest` being a copy of TestOracleIntegrationSmokeTest` and only then  extract common super class)

Reply:
should I squash the 3 commits?

Reply:
It should be either:

2 commits:
- rafactor tests extractin `BaseOracleIntegrationSmokeTest` (still with 1 concrete subclass `TestOracleIntegrationSmokeTest`)
- implement pooling and add `TestOraclePoolIntegrationSmokeTest `

Or just 1 commit which refactors tests, add pooling implementation and `TestOraclePoolIntegrationSmokeTest` alltogether.

Reply:
I guess the `OracleQueryRunner` changes can be treated as a separate commit while other the tests can be a part of Add Oracle connection pool